### ISSUE TITLE
Example for creating a token

### ISSUE THREAD

**Issue by randytorres** [EOSIO/eosjs#139]
Hey James, I'm trying to create a token on the jungle testnet and I'm getting an AssertionError that doesn't make sense to me.

```
AssertionError {name: "AssertionError", actual: undefined, expected: true, operator: "==", message: "Missing currency stats for asset: IOS@eosio.token", …}
```

My Code
```
  maxSupply = "100.0000 IOS"
  async createToken(maxSupply) {
    try {
      const tokenContract = await eos.contract("eosio.token")
      const transaction = await tokenContract.create({
        issuer: "randyt",
        maximum_supply: maxSupply,
      })

      return transaction
    } catch (error) {
      console.error(error)
    }
  }
```

Why would it require currency stats for a token that doesnt exist? I get that it would check to make sure it doesn't exist but i don't see why it would require that if the token is being created. maybe im not understanding the create action correctly?

The example in the readme shows you deploying your own token contract, is that what i need to do?
If not can you please provide an example for creating a token.

**Comment by yankunsam** [comment]
It means that you token did not create successfully.

**Comment by jcalfee** [comment]
It is a bug: https://github.com/EOSIO/eosjs/issues/136

**Comment by jcalfee** [comment]
published eosjs@13.0.1
